src/websrc/json/en.json

```diff
@@ -47,24 +47,24 @@
         "map": "AP"
       },
       "zbc": {
-        "op": "Operating mode",
+        "op": "Mode",
         "opn": "Zigbee-Network",
         "opu": "Zigbee-USB",
-        "scc": "Socket client connected",
+        "scc": "Socket",
         "sccn": "No connection",
-        "sccy": "Yes, {{count}} clients",
+        "sccy": "{{count}} clients",
         "su": "Connection time",
         "tt": "Device status",
         "zfs": "Flash size",
         "zie": "IEEE"
       },
       "c": {
-        "fv": "FW revision",
-        "hv": "HW version"
+        "fv": "Firmware",
+        "hv": "Hardware"
       }
     },
     "ge": {
-      "desc": "General settings",
+      "desc": "Settings",
```

Comment:
All tabs are about settings. No? Maybe we should leave some individual names?

Reply:
Yes, but again, we are ALREADY in the General tab/section, inside we have the basic settings plus related elements with the LED and NTP sections. In other words we have an "General settings" in "General settings"... sounds ugly, 'coz open the General tab, we already know that we are in General settings page

Reply:
You're right that there isn't any sense in "General settings" card in "General settings" page. Maybe we should call this card "Device" or "Network" or even move these field to another page/card?

Reply:
We will get to the point of moving, there is no need to rush things, and this is inevitable. But I propose to leave it this way for now, or based on your proposal, all the functions mentioned on this page actually relate directly to the device, we can rename it to "Device", as u suggested above, so...? It's your turn :smiley:

Reply:
Thank you, let there be a merge. :laughing: